Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1520,6 +1522,11 @@ - (NSString *)cacheKeyForURL:(NSURL *)url processorKey:(NSString *)processorKey
 
 - (void)objectForKey:(NSString *)key options:(PINRemoteImageManagerDownloadOptions)options completion:(void (^)(BOOL found, BOOL valid, PINImage *image, id alternativeRepresentation))completion
 {
+    if ((options & PINRemoteImageManagerDownloadOptionsIgnoreCache) != 0) {
+        completion(NO, NO, nil, nil);
```

Comment:
I think this got missed, the second param here should be YES , otherwise PINRemoteImage will report an error to the client.